app/controllers/assignments_controller.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+# rubocop:disable Metrics/ClassLength
```

Comment:
The preferred way to do this is via `.rubocop.yml`. Even better would be to extract the business logic to a separate file, keep the controller action as simple as possible, and avoid passing the limit. I'm not sure how close to the limit it is already, though.